guides/common/assembly_preparing-environment-for-capsule-installation.adoc

```diff
@@ -17,8 +17,10 @@ endif::[]
 // Supported Operating Systems
 include::modules/ref_supported-operating-systems.adoc[leveloffset=+1]
 
+ifdef::katello,satellite[]
```

Comment:
I'm unsure here. Does satellite imply katello is defined? So can we use:
```suggestion
ifdef::katello[]
```

Reply:
Answered my own question: https://github.com/theforeman/foreman-documentation/pull/680